src/app/child-dev-project/children/children-list/children-list.component.spec.ts

```diff
@@ -87,16 +101,31 @@ describe("ChildrenListComponent", () => {
         Angulartics2Module.forRoot(),
       ],
       providers: [
+        {
```

Comment:
when mocking childrenservice and entitymapperservice we do not need any provider for the database itself